apiary.apib

```diff
@@ -5,6 +5,195 @@ HOST: https://operationcode.org/
+
+        [
+            {
+                "id": 112,
+                "name": "Wyncode Academy",
+                "url": "http://wyncode.co",
+                "logo": "https://raw.githubusercontent.com/OperationCode/operationcode_frontend/master/src/images/codeSchoolLogos/wyncode.jpg",
+                "full_time": true,
+                "hardware_included": false,
+                "has_online": false,
+                "online_only": false,
+                "created_at": "2017-09-21T05:48:50.056Z",
+                "updated_at": "2017-09-21T05:48:50.056Z",
+                "notes": null
+            },
+            {
+                "id": 113,
+                "name": "Nashville Software School",
+                "url": "http://nashvillesoftwareschool.com/",
+                "logo": "https://raw.githubusercontent.com/OperationCode/operationcode_frontend/master/src/images/codeSchoolLogos/nss.jpg",
+                "full_time": true,
+                "hardware_included": false,
+                "has_online": false,
+                "online_only": false,
+                "created_at": "2017-09-21T05:48:50.108Z",
+                "updated_at": "2017-09-21T05:48:50.108Z",
+                "notes": null
+            }
+        ]
+        
+        
+### CodeSchool | Create [/api/v1/code_schools{?name,url,logo,full_time,hardware_included,has_online,online_only,notes}]
++ Parameters 
+  
+  + name (string, required) - The CodeSchool's name
+  + url (string, required) - The CodeSchool's url for homepage
+  + logo (string, required) - The CodeSchool's logo url
+  + full_time (boolean, required) - Are the CodeSchool's courses full time 
+  + hardware_included (boolean, required) - Is Hardware included with the CodeSchool
+  + has_online (boolean, required) - Does the CodeSchool have an online component
+  + online_only (boolean, required) - Is the CodeSchool online only?
+  + notes (text, optional) - Any additional information about the CodeSchool
+
+## Create a CodeSchool Member [POST]
+
+  + Request (application/json)
+
+      + Headers
+
+              Authorization: Bearer Access-Token
+
+      + Body
+      
+                  {
+                      "code_school": {
+                      "name": "CoderSchool",
+                      "url": "http://www.CoderSchool.vn",
+                      "logo": "https://d388w23p6r1vqc.cloudfront.net/img/startups/11852/logo-1457327647.png",
+                      "full_time": "false",
+                      "hardware_included": "false",
+                      "has_online": "true",
+                      "online_only": "false"
+                      }
+                  } 
+
+  + Response 201 (application/json)
+
+      + Body
+
+              {
+                  "code_school": 182
+              }
+
+  + Response 422 (application/json)
+
+          {
+              errors: "Some error message"
+          }
+          
+## CodeSchool | Show [/api/v1/code_schools/{code_school_id}] [GET]
```

Comment:
We'll want an `h3` header below this one, so that it stands alone as its own endpoint.  

The way apiary enforces formatting, we essentially need to have one `h2` and one `h3` for each new endpoint.

Reply:
@hpjaj I get complaints from Apiary when I try to format it the way you suggested. 
I don't have another example of a resource with a `show` & `delete` endpoint to reference on how to get by this. 

Do you know of a workaround? Line `92` & `118`

![screen shot 2017-09-27 at 12 48 03 am](https://user-images.githubusercontent.com/16382165/30875475-dc6df20a-a31d-11e7-9f50-eb14c8ef8a96.jpg)
 so I don't get the semantic issue?

Reply:
For the case where both API endpoints are identical, and have the same parameters, i.e. `[/api/v1/code_schools/{code_school_id}]`, I would: 

- have an `h2` that addresses both (i.e. `## CodeSchool | Show or Delete`)
- have an `h3` for each endpoint, beneath that `h2`

Reply:
@hpjaj thanks for the support on completing this task. Mind taking a look again? I've updated with all of your suggestions.